baseline/OSPS-GV.yaml

```diff
@@ -12,8 +12,17 @@ criteria:
       The project documentation MUST include the
       Roles and Responsibilities for members of the
       project.
-    rationale: # TODO
-    implementation: # TODO
+    rationale: |
+      Documenting project roles and responsibilities
+      helps project particpants, potential contributors,
+      and downstream consumers have an accurate
+      understand of who is working on the project
+      and what areas or authority they may have.
+    implementation: |
+      Documenting project particpants and their roles
+      through such artifacts as members.md, governance.md, 
+      or similar file within the source code repository
+      of the project.
```

Comment:
What about `MAINTAINERS*` files?  It feels like it would be helpful to both sides to have a list of acceptable file locations, rather than the vague "or similar file with the source code repository".  As an example from elsewhere, we expect `LICENSE*` to be at the top-level, though we allow no-suffix or `.txt` as viable suffixes.

We might be able to use https://codelabs.developers.google.com/codelabs/bigquery-github#0 to get some information as to the prevalence of different files, though that won't cover other locations such as websites.

Reply:
oh yeah, forgot that one!  good catch!